doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -0,0 +1,174 @@
+| gbm-kms                        |             |          |            |
+| eglstream-kms                  |             |          |            |
+| eglstream-kms + gbm-kms hybrid |             |          |            |
+| x11                        |             |          |            |
+| wayland                    |             |          |            |
+| virtual                        |             |          |            |
+
+### Platform Selection
+For the following tests, we must test that the compositor can start with the correct
+display platform chosen. **Note that this section should NOT be run for each platform
+by nature**. The remaining test sections _**should**_ be run for each platform
+in the matrix.
+
+1. Test that platforms are selected appropriately
+   - When you are not on an Nvidia platform, verify that `gbm-kms` is selected
+     for each output when the compositor is _not_ running in a hosted environment.
+   - When you have an Nvidia card _and_ you are using Nvidia drivers, then
+     the `eglstream-kms` platform should be selected for that particular output.
+   - When you are hosted by an X compositor, the `x11` platform will be chosen
+
+2. Platforms can be forced to run on a particular display platform if the environment supports it.
+   Platforms are specified with `--platform-display-libs=mir:<platform_name>`
+   - To specify `egl-stream-kms`, you must have an Nvidia card with proprietary drivers
+   - You can force an Nvidia card to use `gbm-kms`
+   - To specify `x11`, you must be on a hosted platform
+   - To specify `wayland`, you must be on a hosted wayland platform
+
+3. Check that the virtual platform can run and you can connect to it via a VNC:
+   ```sh
+   WAYLAND_DISPLAY=wayland-1 miral-app --platform-display-lib=mir:virtual \
+       --virtual-output=1280x1024 \
+       --add-wayland-extension=zwp_virtual_keyboard_manager_v1:zwlr_virtual_pointer_manager_v1:zwlr_screencopy_manager_v1
+   ```
+   After, in a separate VT, connect to the VNC:
+   ```sh
+   WAYLAND_DISPLAY=wayland-1 ubuntu-frame-vnc
+   ```
+   Then, check that you can connect via a VNC viewer:
+   ```sh
+   gvncviewer localhost
+   ```
+
+### Applications
+For each empty box in the matrix above, ensure that the following applications can start
+
+1. Test that the following QT Wayland applications can be started and can receive input:
+    ```sh
+    sudo apt install qtwayland5 kate qterminal
+    
+    # First...
+    kate
+   
+    # Then...
+    qtwayland5
+   
+    # Finally..
+    qterminal
+    ```
+2. Test that `weston-terminal` can be started and can receive input:
+    ```
+    sudo apt install weston
+    weston
+    ```
+3. Test that `glmark2-wayland` can be run:
+    ```sh
+    sudo apt install glmark2-wayland
+    glmark2-wayland
+    ```
+4. Test that `gnome-terminal` can be started and can receive input:
+   ```sh
+   sudo apt install gnome-terminal
+   gnome-terminal
+   ```
+
+### Mir Configuration Options
```

Comment:
This is a bad title. There are lots of Mir configuration options that are not tested here.

It should likely be split into two sections:

### Mir Console Providers

### Window Manager Examples

Note that this section doesn't apply to testing the snaps. (And maybe each suggested section should have a table to check things off)